internal/frontend/assets/html/dashboard.go.html

```diff
@@ -225,7 +227,11 @@ <h2>Sign-in-as</h2>
                 class="button full"
                 type="submit"
               >
-                Impersonate
+                {{if .State.ImpersonateEmail}}
+                  Unimpersonate
```

Comment:
Does this.... work? Do any backend changes need to be made to clear the set status?

Reply:
Yes it seems to work. It sets the `ImpersonateEmail` in the state to the empty string, which is the default unimpersonated state.